src/types/rooms.ts

```diff
@@ -33,7 +33,7 @@ export interface RoomData_v2 {
   title?: string;
   subtitle?: string;
   url?: string;
-  description?: string;
+  about?: string;
```

Comment:
It seems that this change is not supported in other places, isn't it?

Reply:
Description was also an admin v2 thing, and it wasn't used anywhere.